src/admin/class-meta-box.php

```
@@ -90,6 +90,35 @@ function cimo_get_media_type_label( $mimetype ) {
 						$original_filesize = isset( $cimo['originalFilesize'] ) ? floatval( $cimo['originalFilesize'] ) : 0;
 						$converted_filesize = isset( $cimo['convertedFilesize'] ) ? floatval( $cimo['convertedFilesize'] ) : 0;
+
+						$is_bulk_optimized = isset( $cimo['bulk_optimization'] ) &&
+							is_array( $cimo['bulk_optimization'] ) &&
+							! empty( $cimo['bulk_optimization'] );
+
+						// For bulk optimization, we need to recalculate the optimization savings percentage
+						// based on the original and converted file sizes for all the thumbnails
+						if ( $is_bulk_optimized ) {
+							$bulk_optimization_count = count( $cimo['bulk_optimization'] );
+							$bulk_optimization_sizes = array_keys( $cimo['bulk_optimization'] );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$bulk_optimization_sizes'. (undefined)

(UnusedLocalVariable)